### PR TITLE
APPS-2538 Add FAQs Page

### PR DESCRIPTION
**Pull request by bergieboy**
Still waiting on Spanish translations.

### CODE REVIEW

File: src/utils/constants/labels/FAQs.js [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+  ],
+  PROVIDERS: [
+    {
+      Q: {
+        [en]: 'I am a child care provider and I need to know how can I change my telephone number?',
+        [es]: ''
+      },
+      A: {
+        [en]: 'As a child care provider, please contact your assigned Licensed Program Analyst (LPA) '
+        + 'at your local Regional Office and submit changes. ',
```
Comment (anzioka) [inline src/utils/constants/labels/FAQs.js:154]: do you need this trailing space at the end of the string?

File: src/utils/constants/labels/FAQs.js [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+      }
+    },
+    {
+      Q: {
+        [en]: 'I am a child care provider and I need to know how can I change my email address?',
+        [es]: ''
+      },
+      A: {
+        [en]: 'As a child care provider, please contact your assigned Licensed Program Analyst (LPA) at '
+        + 'your local Regional Office and submit changes. ',
```
Comment (anzioka) [inline src/utils/constants/labels/FAQs.js:165]: same question about trailing space

File: src/containers/faqs/FAQsPage.js [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+import { ContentOuterWrapper, ContentWrapper } from '../../components/layout';
+import { getTextFnFromState } from '../../utils/AppUtils';
+import { FAQS, Q_AND_A } from '../../utils/constants/labels';
+
+const { PURPLE } = Colors;
+
+const Wrapper = styled(ContentWrapper)`
+  padding: 30px !important;
+  background-color: white;
```
Comment (anzioka) [inline src/containers/faqs/FAQsPage.js:18]: please sort

File: src/utils/constants/labels/FAQs.js (hunk 1 above)
Comment (bergieboy) [reply src/utils/constants/labels/FAQs.js:154]: nope

**Review by anzioka — APPROVED** [review]

**Review by UnsungHero97 — APPROVED** [review]

**Pull request merged**